source/assets/public/index.css

```diff
@@ -274,26 +293,10 @@ body {
   margin-top: -1rem;
 }
 
-
-
-
 .footer {
   margin-bottom: 6rem;
-  width: 90%;
   text-align: center;
 }
-  .footer__nav {
```

Comment:
moved this to `nav--footer` as most of these styles are shared with the header nav